### ISSUE TITLE
Some questions?

### ISSUE THREAD

**Issue by zhyhy** [ChaofWang/L-GM_loss_pytorch#4]
Hi, thanks for your contribution! But I don't know why computed twice loss ( lgmloss ,and tc_lgm_logits)  in the test_loss? I hope you can answer this question. Thanks a lot.

**Comment by ChaofWang** [comment]
@zhyhy hello，sorry for the doubts. These two are the same, I just want to test whether the behavior of the re-rewritten loss is consistent with the original.
